### DISCUSSION TITLE
Cannot copy YML file to Clipboard

### DISCUSSION THREAD

**Discussion by 1artist** [Lissy93/dashy#1534] [Help and Support]
I just finished installing the latest docker and portainer and dashy onto my fileserver which runs the newest 22.04.4 version of Ubuntu Server OS. After testing out Dashy I find out it does not allow me to copy my YML file to the clipboard. My guess is because my Dashy runs under HTTP and not HTTPS due to ).  However, my Portainer runs under HTTPS because it uses a self signed SSL certificate that got automatically installed without me doing anything. 
How do I configure Dashy so it shares the same Portainer self signed certificates so it can run under HTTPS?  My SSL technical skills are not at an advanced level.  So I would appreciate any advice. At the link below I found the "docker run -d" script below that claims it can pass a Self-Signed Certificate to Dashy:
[https://github.com/Lissy93/dashy/blob/master/docs/management.md#ssl-certificates](url)
```
docker run -d \
  -p 8080:80 \
  -v ~/my-private-key.key:/etc/ssl/certs/dashy-priv.key:ro \
  -v ~/my-public-key.pem:/etc/ssl/certs/dashy-pub.pem:ro \
  lissy93/dashy:latest
```
So, I ran the "sudo docker PS" command below to learn my Portainer app uses ports 8000, 9000 and 9443. Which is not the "8080" port number shown in the script above. In short, can anybody in this forum please help me figure out any changes I need to make to the script above so it fixes my Dashy certificate problem. I want to make sure before I execute the script above it is correctly coded without port number errors.  And please note, I want to avoid using apps like NGINX or Lets Encrypt because as a newbie my goal is to not complicate things anymore and another goal of mine is to keep my Dashy and Portainer fenced in within my home LAN.  In other words, I want to avoid doing any port forwarding of any kind because I want to keep all my apps fenced in within my home LAN and not open to the World Wide Web.  Thank you for your time. 
```
artist@myserver:/$ sudo docker ps
PORTS                                                                                            NAMES
0.0.0.0:4002->80/tcp,:::4002->80/tcp                                                             Dashy
0.0.0.0:8000->8000/tcp, :::8000->8000/tcp, 0.0.0.0:9443->9443/tcp, :::9443->9443/tcp, 9000/tcp   portainer
```

**Comment by CrazyWolf13** [comment]
You cannot use the same as portainer, portainer creates this while installing for itself, I'd recommend watching some youtube tutorials, that's a bit more of a complex are to explain, as this drastically changes on the type of setup YOU have, YOU want to run, and what YOU have.

https://www.youtube.com/watch?v=hExRDVZHhig

Now it depends if you have a bought a public domain, they can be around 8 dollars or even free on some websites.
https://www.youtube.com/watch?v=GarMdDTAZJo

If not, it gets very complex, also importing this will get complicated:
https://www.youtube.com/watch?v=VH4gXcvkmOY
